app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -24,8 +24,11 @@ import org.mozilla.focus.telemetry.TelemetryWrapper
 import org.mozilla.focus.telemetry.UrlTextInputLocation
 import org.mozilla.focus.utils.OnUrlEnteredListener
 import android.graphics.BitmapFactory
+import android.content.Context.MODE_PRIVATE
 
 private const val COL_COUNT = 5
+val BLACKLIST = "blacklist"
```

Comment:
Probably rename this to bundled_blacklist, to be explicit that this is for the bundled tiles.